src/pages/en/guides/markdown-content.mdx

```diff
@@ -133,7 +133,7 @@ description: Find out what makes Astro awesome!
 This is a post written in Markdown.
 ```
 
-Frontmatter properties are then available to the layout through `Astro.props`:
+Metadata are then available to the layout through `Astro.props`. For example, you can access frontmatter properties through `Astro.props.frontmatter`:
```

Comment:
```suggestion
[Specific properties are then available to the layout component](/en/core-concepts/layouts/#markdown-layout-props)  through `Astro.props`. For example, you can access frontmatter properties through `Astro.props.frontmatter`:
```
What about adding a link here instead of another callout, so we can remove the bottom line?

Not entirely opposed to metadata, though in this sentence we'd have some people arguing singular, some plural.  :sweat_smile: I think something like "Specific properties" might be even more useful, since the properties available to a layout component *are* different than properties exported when you're importing a Markdown component otherwise. (We just had this come up re: switching to content collections, which means changing from using a layout property to explicitly importing.)

Reply:
Love this change, thanks!